module-check/src/main/clojure/clojure/core/typed/base_env.clj

```diff
@@ -1445,6 +1445,10 @@ clojure.set/union (All [x] [(Set x) * -> (Set x)])
 clojure.set/intersection (All [x] [(Set x) (Set x) * -> (Set x)])
 clojure.set/difference (All [x] [(Set x) (Set Any) * -> (Set x)])
 
+clojure.repl/apropos [String -> (U (Seq String) nil)]
+clojure.repl/demunge [String -> String]
+clojure.repl/source-fn [clojure.lang.Symbol -> String]
```

Comment:
After Ambrose's comment about `apropos` returning nil, have you checked `demunge` and `source-fn`? I think `source-fn` can also return nil.